serializer/custom_normalizer.rst

```diff
@@ -21,10 +21,10 @@ to customize the normalized data. To do that, leverage the ``ObjectNormalizer``:
 
     use App\Entity\Topic;
     use Symfony\Component\Routing\Generator\UrlGeneratorInterface;
-    use Symfony\Component\Serializer\Normalizer\NormalizerInterface;
+    use Symfony\Component\Serializer\Normalizer\ContextAwareNormalizerInterface;
     use Symfony\Component\Serializer\Normalizer\ObjectNormalizer;
 
-    class TopicNormalizer implements NormalizerInterface
+    class TopicNormalizer implements ContextAwareNormalizerInterface
```

Comment:
If we make this change, we must also change the signature of the function `supportsNormalization` to accept the context in 3rd parameters

Reply:
it's exactly the reason why I propose to change the interface if we want to keep the context parameter in method.

in ContextAwareNormalizerInterface : 

```
public function supportsNormalization($data, $format = null, array $context = []);
```

In NormalizerInterface:

```    
public function supportsNormalization($data, $format = null);
```

If we want keep `NormalizerInterface` we need to remove the context in the example.

Reply:
Target change (4.1 to 4.3)